changelog/pending/20230609--sdkgen-python--fixes-python-always-printing-input-deprecation-messages.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: sdkgen/python
+  description: Fixes python always printing input deprecation messages.
```

Comment:
This PR is more easily reviewed by navigating to the commits tab and then viewing the second commit.